crates/blockchain/error.rs

```diff
@@ -80,6 +80,8 @@ pub enum MempoolError {
     NotEnoughBalance,
     #[error("Transaction gas fields are invalid")]
     InvalidTxGasvalues,
+    #[error("Transaction sender is invalid")]
+    InvalidTxSender(String),
```

Comment:
We can define it as `InvalidTxSender(#[from] secp256k1::Error)` that way we would't have to explicitly convert to it (we can rely on ? to do it for us) and can leverage the crate's error messages.

Reply:
We should also include the content on the error message, we can do so by adding `{0}` to the error message, such as "Transaction sender is invalid: {0}".